systems/plants/@RigidBodyManipulator/forwardKin.m

```diff
@@ -1,4 +1,4 @@
-function [x,J,dJ] = forwardKin(obj,kinsol,body_or_frame_ind,pts,rotation_type, base_or_frame_ind)
+function [x,J,dJ] = forwardKin(obj,kinsol,body_or_frame_ind,pts,rotation_type, base_or_frame_ind, options)
```

Comment:
rotation_type is in the options structure, but I didn't want to change existing functionality yet. If you think we should I can change it.
Should I also change body_or_frame_ind to body_or_frame_id then?
My thoughts were to have base_or_frame_ind not be an option but an actual argument just like body_or_frame_ind. I'm not advocating this, but why not move body_or_frame_ind to the options struct too then?